spec/models/user_spec.rb

```diff
@@ -1087,4 +1087,25 @@
     end
   end
 
+  describe ".info_requests_count" do
```

Comment:
ah, that was more a sanity check that the underlying functionality didn't get broken by the patch